app/models/miq_event_definition_set.rb

```diff
@@ -3,8 +3,8 @@ class MiqEventDefinitionSet < ApplicationRecord
 
   def self.seed
     existing = all.group_by(&:name)
-    CSV.foreach(fixture_path, :headers => true, :skip_lines => /^#/) do |csv_row|
-      set = csv_row.to_hash
+    YAML.load_file(fixture_path).each do |c|
+      set = c.to_hash
```

Comment:
Since the yaml file is already in hash format I think you can get rid of the .to_hash bit

```suggestion
    YAML.load_file(fixture_path).each do |set|
```